db/app.js

```diff
@@ -4,20 +4,27 @@ app.use(express.json());
 const {
     apiHealthCheck,
     getTopics,
-    getArticleById } = require("./controller")
+    getArticleById, 
+    getArticles } = require("./controller")
 app.get('/api', apiHealthCheck)
 
 app.get('/api/topics', getTopics)
 
 app.get('/api/articles/:article_id', getArticleById)
 
+app.get('/api/articles', getArticles)
+
 app.use((err, req, res, next) => {
-    console.log(err.msg)
     if (err.status) {
         res.status(err.status).send({ msg: err.msg })
     } else {
         res.status(500).send({ msg: 'Internal Server Error' });
     }
 })
 
+app.use((req, res, next) => {
+    res.status(404).send({ msg: 'Route not found' });
+  });
```

Comment:
This will work as long as it's always at the bottom... Is there a refactor we could do so that it will always work regardless?